src/quacc/recipes/espresso/core.py

```diff
@@ -148,6 +152,91 @@ def relax_job(
     )
 
 
+@job
+def external_relax_job(
```

Comment:
Elsewhere, I had been calling this `ase_relax_job` but perhaps your naming is clearer. I'll let you decide.